<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setDateDlg</class>
 <widget class="QDialog" name="setDateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbtnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>242</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>248</width>
     <height>197</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnOk">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
